backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/calendar/BpkCalendarController.kt

```diff
@@ -90,9 +90,12 @@ class BpkCalendarController(
         }
     }
 
-    internal val firstVisibleItemMonth: YearMonth
+    private val firstVisibleItemMonth: YearMonth
         get() = _state.cells[lazyGridState.firstVisibleItemIndex].yearMonth
 
+    internal val visibleMonths: Set<YearMonth>
+        get() = lazyGridState.layoutInfo.visibleItemsInfo.map { _state.cells[it.index].yearMonth }.toSet()
```

Comment:
Nice change @henrik-sky 
So now instead of passing only one month, you pass all visible months, curios to know what happens when a user scrolls by just half a month, what this callback will send?